manifests/8/8212369/WPR/α0.01/8212369.WPR.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: 8212369.WPR
+PackageVersion: α0.01

```

```suggestion
PackageVersion: 0.0.1-alpha
```
Prefixes on versions do not work well, and unicode characters should be avoided as they do not work well with scripts in many cases